### PR TITLE
feat: CRUD API's

### LINKED ISSUE #2: [stencil] CRUD APIs out of the box
## ToDo: 

- [ ] Add support for having support for CRUD APIs out of box. 

## References: 
- https://docs.nestjs.com/recipes/crud-generator
- https://github.com/nestjsx/crud

### PR DESCRIPTION
**Pull request by Apurv428**
Created generic crud APIs

Issue: https://github.com/SamagraX-Stencil/stencil/issues/2

### CODE REVIEW

File: sample/01-all-features/src/generic/generic.controller.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import { Body, Controller, Delete, Get, Param, Post, Put, Req, Res } from "@nestjs/common";
+import { Request, Response } from "express";
```
Comment (techsavvyash) [inline sample/01-all-features/src/generic/generic.controller.ts:2]: stencil is fastify based.

File: sample/01-all-features/src/generic/generic.model.ts
Comment (techsavvyash) [inline sample/01-all-features/src/generic/generic.model.ts:1]: i think this file was committed by mistake

**Review by techsavvyash — CHANGES_REQUESTED** [review]
I think we already have the logic to generate generic services and modules and even controllers in the schematics already. 
The extractor is a promising thing to add but I think the crude logic will not be very maintainable, let's use this [prisma-parser](https://github.com/techsavvyash/pp) to parse the `schema.prism` files in a JSON model and then use it to add types.
Also, I believe the code for this is best added in the [schematics](https://github.com/samagrax-stencil/schematics) since all the code generation and injection logic is consolidated there for the `stencil-cli` to use.

**Comment by Savio629** [comment]
// @Apurv428

**Comment by techsavvyash** [comment]
Closing this since there is no movement.

**Pull request closed without merging**